[DTM] Asset location (#1450)
SoftwareAG/c8y-docs: content/dtm/asset-types-bundle/asset-properties.md

Asset properties are the parameters that define an [asset model](#asset-models). One or more asset properties are used to define an asset model. Asset properties are created in the **Asset properties** page.

> Default properties are available once you login into the tenant. See [Default properties](/dtm/default-properties/#characteristics-of-default-properties) to know the list of properties that are available.

Author: Modified

Use the downloaded file to import the asset properties into a DTM application on another tenant.

{{< c8y-admon-info>}}

> Default properties are not listed under Export asset properties. See [Default properties](/dtm/default-properties/#characteristics-of-default-properties) to view the list of properties available.

Author: Formatted Export asset properties button to bold

> Default properties are not listed under Export asset properties. See [Default properties](/dtm/default-properties/#characteristics-of-default-properties) to know the list of properties that are available.

Author: Modified

2. Upload the JSON file in the drop file area of the resulting dialog. The JSON file is the file containing exported asset properties from a DTM application on another tenant.
3. Click **Next** to preview the asset properties before importing them.
4. Click **Import** to import the asset properties.

> 5. Alternatively, Click **Back** to go to previous step. Hover over the file name and click the **Delete** icon to remove the current file and upload a different one.

Author: Modified

You can import a maximum of 500 asset properties at a time.
{{</ c8y-admon-info>}}

### Default properties {#default-properties}

### Characteristics {#characteristics-of-default-properties}

* Default properties are readily available when you open the application.
* Default properties cannot be deleted. 
* Only selected fields within a default property are editable and the editable fields vary depending on the specific property.
* Default properties cannot be exported.

### Location {#location}

> Location property enables you to assign location (latitude and longitude) to an asset. You can set the default values of latitude and longitude by entering corresponding values or alternatively using the map view.

Reviewer: ```suggestion
The **Location** property enables you to assign location (latitude and longitude) to an asset. You can set the default values of the latitude and longitude by entering the corresponding values or alternatively using the map view.
```

Author: Modified

> To set the default values for latitude and longitude using map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. The marker is positioned at D&uuml;sseldorf, Germany by default. Click or drag the marker to the preferred position to select the default value for latitude and longitude. To close the map view, click **Close** above the map.

Reviewer: ```suggestion
To set the default values for the latitude and longitude using the map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. The marker is positioned at Duesseldorf, Germany by default. Click or drag the marker to the preferred position to select the default value for latitude and longitude.
```

Author: @MWindel is it not necessary to add Umlaut for Duesseldorf?

{{<c8y-admon-info>}}

> The valid range for the latitude is -90 to +90 and valid range for the longitude is -180 to +180. <br>

Reviewer: ```suggestion
The valid range for the latitude is -90° to +90° and valid range for the longitude is -180° to +180°. <br>
```

Author: Latitude and Longitude are fields which accepts both negative and position numbers within that range. 
@MWindel Should I still modify that

> The valid range for Latitude is -90 to +90 and valid range for Longitude is -180 to +180. <br>

Reviewer: ```suggestion
The valid range for the latitude is -90 to +90 and valid range for the longitude is -180 to +180. <br>
```

Author: Modified

> The marker on the map is positioned at an approximated location when either latitude or longitude value is missing.<br>

Reviewer: ```suggestion
The marker on the map is positioned at an approximated location when either the latitude or longitude value is missing.<br>
```

Author: Modified

> The default values of latitude and longitude are automatically updated whenever a new location is selected on the map and vice-versa. The Altitude value is not represented on the map.

Reviewer: ```suggestion
The default values of both latitude and longitude are automatically updated whenever a new location is selected on the map and vice-versa. The Altitude value is not represented on the map.
```

Author: Modified
